### DISCUSSION TITLE
SEO settings

### DISCUSSION THREAD

**Discussion by jeromeatneotek** [givanz/Vvveb#87] [General]
using pgsql I have added the SEO plugin and when I try and go to settings I get a 404 and the error below
C:\Domains\myweb\wwwroot\public\plugins\seo\admin\settings.html on line.

Perhaps there are no settins required for SEO and so this menu item should not be there?

**Comment by givanz** [comment]
Sorry for this, the file is now included in latest.zip

The SEO plugin is under active development and a complete version will be available in the following days currently it has many missing features.

**Reply by jeromeatneotek** [reply]
I can see that now, looks great, will you be making the SEO plugin repository available?

**Reply by givanz** [reply]
Thanks, structured data schema will need more time as there are many formats to support.

Yes, currently it is bundled with the zip but once complete it will be moved to it's own repository like other plugins https://github.com/topics/vvveb-plugin
